message.xml

```diff
@@ -6,5 +6,11 @@ this repository contains the full copyright notices and license terms. -->
     <record model="ir.message" id="msg_beneficiaris_percent">
         <field name="text">Beneficiaris of Crop "%(crop)s" and Plantation "%(plantation)s" must sum 100.</field>
     </record>
+    <record model="ir.message" id="msg_vintage_limit">
+        <field name="text">This product cannot have more than one Vintage because of its agronomic type.</field>
```

Comment:
Messages always must indicate to which record we're referring to. Ie:

Product "%(product)s" cannot have more than one Vintage because of its agronomic type.